lang/languages.xml

```diff
@@ -30,7 +30,7 @@ List of available languages for your wallet's seed:
         <language display_name="Italiano" locale="it_IT" wallet_language="Italiano" flag="/lang/flags/italy.png" qs="none"/>
         <language display_name="日本語" locale="ja_JP" wallet_language="日本語" flag="/lang/flags/japan.png" qs="none"/>
         <language display_name="Nederlands" locale="nl_NL" wallet_language="Nederlands" flag="/lang/flags/netherlands.png" qs="none"/>
-        <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/> -->
+        <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/>
```

Comment:
Please double check that this doesn't break the layout on the language selection screen.

Reply:
![Screenshot from Monero GUI language selection page](https://user-images.githubusercontent.com/24587734/36795743-182d6bae-1ca4-11e8-913c-4fc0987776aa.jpg)
Freshly built.

Reply:
Thanks!